changelog/unreleased/bugfix-ocswitch-off-state-is-not-pixel-perfect

```diff
@@ -0,0 +1,5 @@
+Bugfix: OcSwitch Off State is not pixel perfect
+
+The off state of the OcSwitch was modified by 1px.
```

Comment:
Please don't state what can be read from the code below, changelog items are supposed to explain what your changes fixed (e.g. "We've addressed the fact that the OcSwitch wasn't aligned properly in it's off state")